tools/salmon/macros.xml

```diff
@@ -12,11 +12,48 @@
             <requirement type="package" version="1.4.1">scipy</requirement>
         </requirements>
     </xml>
+
+    <xml name="geometry">
```

Comment:
this is the only new parameter I would include ... can you work on this? Currently, this is not complete not is it actually used.

Reply:
Sure!